VK/VK/FriendListScreen/FriendListTableViewController.swift

```diff
@@ -7,36 +7,29 @@ typealias TapHandler = (NetworkUnit) -> ()
 
 /// Экран со списком друзей.
 final class FriendListTableViewController: UITableViewController, UIViewControllerTransitioningDelegate {
+    // MARK: - Private constants
+
+    private struct Constants {
+        static let fieldsValue = "city, photo_200_orig"
+        static let fieldsName = "fields"
+        static let emptyCharacter = Character(" ")
```

Comment:
```suggestion
        static let emptyString = ""
```
ты дальше из character string делаешь, почему сразу не делать string

Reply:
Мне изначально нужен был именно char, но в целом согласен. Добавил отдельно пустую строку